src/pages/Dashboard.js

```diff
@@ -286,85 +300,124 @@ const DashBoardComponent = () => {
       </FirstRow>
 
       {/* Doctors View - Patients List along with current conditions */}
-      <PatientsWrapper isInitLoading={isInitLoading}>
-        {patients?.length > 0
-          ? patients?.map((patient) => (
-              <PatientCard key={patient.patientId} patient={patient} />
-            ))
-          : searchText?.length > 0 && (
-              <NoPatientsWrapper>
-                <p>
-                  <strong>No results found</strong>
-                </p>
-                <Button onClick={clearSearchInput} className="link-button">
-                  Back to dashboard
-                </Button>
-              </NoPatientsWrapper>
+      {isMobile ? (
```

Comment:
Please make the same way it is done in Patients.js: move mobile and desktop code in separate components and leave in return clear condition.
![image](https://user-images.githubusercontent.com/13137535/121342473-5366fd80-c92a-11eb-86c0-73d6a2f225ea.png)